Tests/SpotHeroAPITests/TestCases/RequestTests/SearchGetMonthlyFacilitiesRequestTests.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
+
+@testable import SpotHeroAPINext
+import XCTest
+
+private protocol SearchGetMonthlyFacilitiesRequestTests: APITestCase {
+    func testGetMonthlyFacilitiesSucceeds()
+}
+
+private extension SearchGetMonthlyFacilitiesRequestTests {
+    /// Attempts to fetch a list of monthly facilities, expecting success
+    func getMonthlyFacilities(latitude: Double, longitude: Double, startDate: Date, file: StaticString = #file, line: UInt = #line) {
+        let request = SearchGetMonthlyFacilitiesRequest(client: Self.newNetworkClient(for: .craig))
+        let expectation = self.expectation(description: "Fetched monthly facilities.")
+        
+        let parameters = SearchGetMonthlyFacilitiesRequest.Parameters(latitude: latitude,
+                                                                      longitude: longitude,
+                                                                      startDate: startDate,
+                                                                      pageSize: 1)
+        
+        request(parameters: parameters) { result in
```

Comment:
Should this have a `self.`?
```suggestion
        self.request(parameters: parameters) { result in
```

Reply:
Nope, the request is created on line 13.

Reply:
Oh whoops, I think the highlighting threw me off